doc/man/8/rbd.rst

```diff
@@ -120,7 +120,8 @@ Parameters
 
 .. option:: --pretty-format
 
-   Make json or xml formatted output more human-readable.
+   Make json or xml formatted output more human-readable, and it only works
```

Comment:
Nit: seems redundant to me since it already states it's for making json or xml output.